CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ as necessary. Empty sections will not end in the release notes.
 
 ### New Features
 
+- Nessie's persistence layer now supports extensible object types. This allows to store objects that
```

Comment:
It's an internal change - nothing user facing - no need for a changelog entry IMO